internal/pkg/describe/env.go

```diff
@@ -25,10 +35,13 @@ type EnvDescription struct {
 // EnvDescriber retrieves information about an environment.
 type EnvDescriber struct {
 	env  *archer.Environment
+	proj *archer.Project
 	apps []*archer.Application
 
-	store        storeSvc
-	sessProvider *sess.Session
+	store          storeSvc
+	sessProvider   *sess.Session
+	rgClient       resourcegroups.ResourceGroupsClient
+	cloudformation *cloudformation.CloudFormation
```

Comment:
Seems like `sessProvider` and `cloudformation` are defined but not used?

Reply:
Ah, right--now that I took out the CFN Describe method. Thx!